### PR TITLE
Azure Lab Services Python sample for April 2022 update

### PR DESCRIPTION
**Pull request by RogerBestMsft**

### CODE REVIEW

**Comment by RogerBestMsft** [comment]
https://github.com/Azure-Samples/azure-samples-python-management/issues/97

**Comment by RogerBestMsft** [comment]
@msyyc Can I get a review?

**Comment by msyyc** [comment]
Hi @BigCat20196 Please help review the PR

**Review by BigCat20196 — APPROVED** [review]

File: samples/labservices/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+export AZURE_CLIENT_ID="xxx"
+export AZURE_CLIENT_SECRET="xxx"
+export SUBSCRIPTION_ID="xxx"
+```
+
+### Installation
+
+1.  If you don't already have it, [install Python](https://www.python.org/downloads/).
+
+    This sample (and the SDK) is compatible with Python 2.7, 3.3, 3.4, 3.5 and 3.6.
```
Comment (BigCat20196) [inline samples/labservices/README.md:41]:
```suggestion
    This sample (and the SDK) is compatible with Python 3.6+.

```

The current SDK version requires python3.6 at least.

**Comment by RogerBestMsft** [comment]
@BigCat20196, can you merge this?

**Pull request merged**